### PR TITLE
BPF CO-RE Changes

### PR DESCRIPTION
**Pull request by Atul-source**
In this PR  I have made the changes to eBPF Programs to support BPF CO-RE (Compile Once Run Everywhere).
This PR address #28 .
PR for Payload Changes: [110](https://github.com/l3af-project/l3af-arch/pull/110)

### CODE REVIEW

File: ipfix-flow-exporter/bpf_ipfix_kern_common.h [hunk 1]
```diff
@@ -32,6 +32,18 @@ typedef struct flow_record_ {
     u16 counter;                    /* flow_idle_counter */
 } flow_record_t;
 
+void *memset(void *b, int c, unsigned long len)
```
Comment (pmoroney) [inline ipfix-flow-exporter/bpf_ipfix_kern_common.h:35]: Why are we adding this? I don't see a reference to it

**Review by pmoroney — APPROVED** [review]
LGTM, just one quick question

File: ipfix-flow-exporter/bpf_ipfix_kern_common.h (hunk 1 above)
Comment (Atul-source) [reply ipfix-flow-exporter/bpf_ipfix_kern_common.h:35]: This Function is Present in <string.h> which we are not including hence I have implemented this function, when I am including <string.h> in our bpf Program it is creating lot of compilation issues.

**Comment by sanfern** [comment]
Can we keep connection limit code to work in window env, without BPF-CORE. Need to support both modes

**Review by sanfern — APPROVED** [review]
LGTM

**Pull request merged**